### ISSUE TITLE
Intellisense not working properly.

### ISSUE THREAD

**Issue by babul7154** [microsoft/python-language-server#1976]
## Environment data

- Language Server version: 0.5.45.0
- OS and version: window 10 version 1909, os build-18363.720
- Python version (& distribution if applicable, e.g. Anaconda): 3.7.7(Anaconda)
- Type of virtual environment used: conda
- Python Language Server version : 0.5.45.0
- VS Code version:Version: 1.43.2 (user setup)
Commit: 0ba0ca52957102ca3527cf479571617f0de6ed50
Date: 2020-03-24T07:38:38.248Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

## Expected behaviour

Intellisense suggests print,asarray,array,arange function.

## Actual behaviour

Intelli-sense is not suggesting print,array,asarray,arange and many more built-in function.

## Logs

```
XXX
```

## Code Snippet / Additional lnformation

```python
XXX
```

**Comment by babul7154** [comment]
When I change enviornment from python 3.7.7(Anaconda) to python 3.7.3 it works fine,so i think It is caused by python version.

**Comment by jakebailey** [comment]
`XXX` aren't logs; please follow the troubleshooting guide / template to give us the verbose logs which will contain info we need to see what's going on. https://github.com/microsoft/python-language-server/blob/master/TROUBLESHOOTING.md#filing-an-issue

**Comment by babul7154** [comment]
sorry but i dont know how to produce vs code log file.

**Comment by jakebailey** [comment]
Have you looked at the link I provided? You have to enable a config, reload VS Code, then look at the Python Language Server output window.

**Comment by babul7154** [comment]
I sort out this problem by deleting vscode intellicode folder(C:\Users\My World\.vscode\extensions\visualstudioexptteam.vscodeintellicode-1.2.6) .The problem was not due to Python version but was due to cache of vscode intellicode.I reinstall Intellicode extension
 and everything works fine now.But microsoft should find a proper way to solve this issue.Sorry for my bad English.
![1](https://user-images.githubusercontent.com/46845524/77906006-d4495480-72a4-11ea-8026-6046860c1da1.PNG)
![2](https://user-images.githubusercontent.com/46845524/77906011-d6131800-72a4-11ea-82bf-f399a959784a.png)

**Comment by jakebailey** [comment]
You seem to have deleted the comment (I got an email, but it's gone), but if the fix was actually to delete IntelliCode and reinstall it, you may want to report an issue on their tracker. https://github.com/MicrosoftDocs/intellicode

**Comment by babul7154** [comment]
Yes everything was working fine after I deleted vscode intellicode  extension folder from **C:\Users\My World\.vscode\extensions\visualstudioexptteam.vscodeintellicode-1.2.6** then reinstalling VS Intellicode  extension  and It was working smoothly but after few hours of my coding same problem arose.Sometime it works as expected,sometimes it doesn't work .And I delete my previous comment because I think i sort out my problem.I am going to file this issue on https://github.com/MicrosoftDocs/intellicode. Sorry for my bad English.

> You seem to have deleted the comment (I got an email, but it's gone), but if the fix was actually to delete IntelliCode and reinstall it, you may want to report an issue on their tracker. https://github.com/MicrosoftDocs/intellicode

**Comment by babul7154** [comment]
One thing can I point out this issue on https://github.com/MicrosoftDocs/intellicode.

**Comment by jakebailey** [comment]
You may want to cross link it, yes.
